config/setup/log.go

```diff
@@ -20,12 +22,17 @@ func Log(c *Controller) (middleware.Middleware, error) {
 	c.Startup = append(c.Startup, func() error {
 		for i := 0; i < len(rules); i++ {
 			var err error
-			var file *os.File
+			var file io.Writer
 
 			if rules[i].OutputFile == "stdout" {
 				file = os.Stdout
 			} else if rules[i].OutputFile == "stderr" {
 				file = os.Stderr
+			} else if rules[i].OutputFile == "syslog" {
+				file, err = gsyslog.NewLogger(gsyslog.LOG_NOTICE, "SYSLOG", "caddy")
```

Comment:
I made mistake here, I thought NOTICE was before INFO but it is the other way around. I think it should be LOG_INFO.